src/thirdparty/axom/fmt/args.h

```diff
@@ -62,12 +62,12 @@ class dynamic_arg_list {
 
 /**
   \rst
-  A dynamic version of `fmt::format_arg_store`.
+  A dynamic version of `axom::fmt::format_arg_store`.
   It's equipped with a storage to potentially temporary objects which lifetimes
   could be shorter than the format arguments object.
 
-  It can be implicitly converted into `~fmt::basic_format_args` for passing
-  into type-erased formatting functions such as `~fmt::vformat`.
+  It can be implicitly converted into `~axom::fmt::basic_format_args` for passing
+  into type-erased formatting functions such as `~axom::fmt::vformat`.
```

Comment:
I know this is a search-and-replace refactor, but what are those tilde characters doing?  Are they destructors?  Is this a corner of markdown I have been blind to?

Reply:
Good point, @agcapps

Reply:
I'm not sure but it originated from fmt.  I think its a typo because i dont think a destructor makes sense there.